README.md

```diff
@@ -13,12 +13,13 @@ See also: https://github.com/pre-commit/pre-commit
 
 Add this to your `.pre-commit-config.yaml`
 
+```yaml
     -   repo: https://github.com/pre-commit/pre-commit-hooks
         rev: v2.5.0  # Use the ref you want to point at
         hooks:
         -   id: trailing-whitespace
         # -   id: ...
-
+```
```

Comment:
the indentation needs fixing

Reply:
Thank you for pointing out. Fixed. Is this okay?